src/api/views.py

```diff
@@ -20,6 +20,7 @@
 from api.serializers import ValidateSerializer,ConvertSerializer,CompareSerializer,CheckLicenseSerializer,SubmitLicenseSerializer,ValidateSerializerReturn,ConvertSerializerReturn,CompareSerializerReturn,CheckLicenseSerializerReturn,SubmitLicenseSerializerReturn
```

Comment:
Don't force push commits. Makes it hard to review changes. We can squash commits while merging

Reply:
Alright. I won't force push commits from now onwards.